src/main/java/formflow/library/ScreenController.java

```diff
@@ -39,634 +54,634 @@
 @RequestMapping("/flow")
 public class ScreenController extends FormFlowController {
 
-  private final List<FlowConfiguration> flowConfigurations;
-  private final ValidationService validationService;
-
-  private final AddressValidationService addressValidationService;
-
-  private final ConditionManager conditionManager;
-  private final ActionManager actionManager;
-
-  public ScreenController(
-      List<FlowConfiguration> flowConfigurations,
-      SubmissionRepositoryService submissionRepositoryService,
-      ValidationService validationService,
-      AddressValidationService addressValidationService,
-      ConditionManager conditionManager,
-      ActionManager actionManager) {
-    super(submissionRepositoryService);
-    this.flowConfigurations = flowConfigurations;
-    this.validationService = validationService;
-    this.addressValidationService = addressValidationService;
-    this.conditionManager = conditionManager;
-    this.actionManager = actionManager;
-    log.info("Screen Controller Created!");
-  }
-
-  /**
-   * Chooses which screen template and model data to render.
-   *
-   * @param flow        The current flow name, not null
-   * @param screen      The current screen name in the flow, not null
-   * @param uuid        The uuid of a subflow entry, can be null
-   * @param httpSession The current httpSession, not null
-   * @return the screen template with model data
-   */
-  @GetMapping("{flow}/{screen}")
-  ModelAndView getScreen(
-      @PathVariable String flow,
-      @PathVariable String screen,
-      @RequestParam(required = false) Map<String, String> query_params,
-      @RequestParam(value = "uuid", required = false) String uuid,
-      HttpSession httpSession
-  ) throws NoHandlerFoundException {
-    log.info("getScreen: flow: " + flow + ", screen: " + screen);
-
-    var currentScreen = getScreenConfig(flow, screen);
-    Submission submission = submissionRepositoryService.findOrCreate(httpSession);
-    if ((submission.getUrlParams() != null) && (!submission.getUrlParams().isEmpty())) {
-      submission.mergeUrlParamsWithData(query_params);
-    } else {
-      submission.setUrlParams(query_params);
+    private final List<FlowConfiguration> flowConfigurations;
```

Comment:
Weird... Looks like it went to 4 spaces for tabs now.  Can you revert that?

Reply:
Shouldn't the `PlatformFlavoredGoogleStyle` we all use apply this identically?
<img width="302" alt="Screenshot 2023-07-06 at 2 19 09 PM" src="https://github.com/codeforamerica/form-flow/assets/4923102/e9498752-11d4-4f64-87a5-ebeec927ce14">

Reply:
weird, I thought so? 
<img width="634" alt="Screenshot 2023-07-06 at 5 43 38 PM" src="https://github.com/codeforamerica/form-flow/assets/7966375/d86eec42-4303-4190-a515-8d3b879fe6fb">

Reply:
Not sure why yours is using 4 - the xml in our repo uses 2: https://github.com/codeforamerica/form-flow/blob/main/intellij-settings/PlatformFlavoredGoogleStyle.xml

Reply:
Huh my bad I reimported it and it set properly, no idea how. Submitting a change to set that back.